ee/tabby-webserver/src/schema/email.rs

```diff
@@ -16,6 +44,9 @@ pub trait EmailService: Send + Sync {
         smtp_username: String,
         smtp_password: Option<String>,
         smtp_server: String,
+        from_address: String,
+        encryption: Encryption,
+        auth_method: AuthMethod,
```

Comment:
```suggestion
        input: EmailSettingInput
```

Please follow the example in https://github.com/TabbyML/tabby/pull/1419 on `*Input`